context/app/static/js/components/Detail/Visualization/Visualization.jsx

```diff
@@ -1,22 +1,83 @@
-import React from 'react';
-import Paper from '@material-ui/core/Paper';
+import React, { useState, useEffect } from 'react';
 import { Vitessce } from 'vitessce';
+import Paper from '@material-ui/core/Paper';
+import Link from '@material-ui/core/Link';
+import ZoomOutMapIcon from '@material-ui/icons/ZoomOutMap';
 import SectionHeader from '../SectionHeader';
-import SectionContainer from '../SectionContainer';
-import 'vitessce/build-lib/es/production/static/css/index.css';
+import {
+  vitessceFixedHeight,
+  StyledSectionContainer,
+  ExpandButton,
+  TopSnackbar,
+  ExpandableDiv,
+  FlexContainer,
+  FlexBottomRight,
+  StyledFooterText,
+} from './style';
+import 'vitessce/dist/es/production/static/css/index.css';
 
 function Visualization(props) {
   const { vitData } = props;
 
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSnackbarOpen, setIsSnackbarOpen] = useState(false);
+
+  function handleExpand() {
+    setIsExpanded(true);
+    setIsSnackbarOpen(true);
+  }
+
+  function handleCollapse() {
+    setIsExpanded(false);
+    setIsSnackbarOpen(false);
+  }
```

Comment:
Do we need both `setIsExpanded` and `setIsSnackBarOpen` if their values are in-sync?

Reply:
Their values are not always in sync, the Snackbar component sets isSnackbarOpen to `false` after 4 seconds when it disappears.